src/firmware_new/proto/control.proto

```diff
@@ -0,0 +1,27 @@
+syntax = "proto2";
+
+// A test robot msg which serves as an example to modify and use to
```

Comment:
Is there an issue to properly define these messages?

Reply:
no there isn't, I think that will just come w/ whoever is going to use the `RobotCommunicator` class, the control msg is just a test msg like it says

Reply:
Fine